specs/slider/slider.md

```diff
@@ -0,0 +1,209 @@
+
+- *A designer customizing the component.*
+A designer can override any internal styling applied to the slotted labels, the background track, the slider track and the thumb(s) if appropriate.
+  
+### Features
+- Implement your own lower and upper thumb UI, default provided.
+- Implement your own background and slider track UI, default provided.
+- Developers can hook up to the `slider`'s `onValueChange` callback to be notified when the value has changed.
+- Customize the tick marks along the track.
+- Form association, if the `slider` component is part of a form the results will be submitted along with the form.
+
+### Prior Art/Examples
+- [FAST-DNA (React)](https://explore.fast.design/components/slider)
+- [Material UI](https://material-ui.com/components/slider/)
+
+---
+
+### API
+
+*Component Name*:
+- `fast-slider`
+
+*Props/Attrs*
+- `min` - minimum allowed value for the slider
+- `max` - maximum allowed value for the slider
+- `step` - limits the values of the `slider` to increments of the `step` value added to the minimum value of the 
+`slider`'s total range.  The default value is 1. The minimum and maximum values of a `slider`'s range are always valid results regardless of the `step` prop. The `step` prop is used as the value for incrementing the thumb by pressing the arrow keys.
+- `mode`
+The slider component can be set to different modes using the `mode` prop.  The default is `singleValue`.  The modes are as follows:
+ > - `singleValue` -  A single thumb, no selected range is displayed.
+ > - `adjustUpperValue` -  A single thumb associated with the upper value of slider's selected range.  The *slider*'s front bar element displays and is anchored to the low end of the slider's current range as the user adjusts the maximum value.
+ > - `adjustLowerValue` -  A single thumb associated with the lower value of slider's selected range.  The *slider*'s front bar element displays and is anchored to the high end of the slider's current range as the user adjusts the minimum value.
+ > - `adjustBoth` - in which case the selected range starts at 40% to 60% of the total range
+
+ - `initialValue` - The `initialValue` prop allows authors to specify the initial selected range of the `slider`.  It defaults to a (step constrained) value at the midpoint on the `slider`'s total range for all modes except `adjustBoth`.
+
+The aria values by default will be the min, max and value number but can be overridden via these props
+- `aria-valuemin` -
+- `aria-valuemax` - 
+- `aria-valuenow` - 
+
+*Events*
+- `onValueChange`
+
+*Slots*
+- `backgroundTrack`
+- `foregroundTrack`
+- `sliderTrack`
+- `lowerThumb`
+- `upperThumb`
+
+
+### Anatomy and Appearance
+**Structure:**
+
+```
+  <div
+    role="slider"
+    $aria-valuenow="${x => x.aria-valuenow}"
+    $aria-valuemin="${x => x.aria-valuemin}"
+    $aria-valuemax="${x => x.aria-valuemax}"
+  >

```

Comment:
We'll want these aria attributes on the host element, rather than on the root element of the div. As mentioned above, no need to have aria properties as well. We can derive the aria attribute value from the normal property values and replicate that to the host.